src/OmniSharp.MSBuild/ProjectLoader.cs

```diff
@@ -115,10 +116,37 @@ public MSB.Evaluation.Project EvaluateProjectFile(string filePath)
             }
         }
 
-        private MSB.Evaluation.Project EvaluateProjectFileCore(string filePath)
+        private MSB.Evaluation.Project EvaluateProjectFileCore(string filePath, Dictionary<string, string> projectConfigurationsInSolution = null)
         {
+            var localProperties = new Dictionary<string, string>(_globalProperties);
+            if (projectConfigurationsInSolution != null
+                && localProperties.TryGetValue(PropertyNames.Configuration, out string solutionConfiguration))
+            {
+                if (!localProperties.TryGetValue(PropertyNames.Platform, out string solutionPlatform))
+                {
+                    solutionPlatform = "Any CPU";
```

Comment:
I wonder if there are any other alternate values for Any CPU we should be aware of?